app/templates/screwdriver.yaml

```diff
@@ -0,0 +1,6 @@
+jobs:
+  main:
+    image: node:6
+    steps:
+      - init: npm install
+      - test: npm test
```

Comment:
@petey this screwdriver.yaml is in `app/templates`. There is another one in
https://github.com/screwdriver-cd/generator-screwdriver/blob/master/screwdriver.yaml, which is what the pipeline picks up I think.